pkg/reconciler/kubernetes/tektonhub/tektonhub.go

```diff
@@ -224,6 +224,25 @@ func (r *Reconciler) manageApiComponent(ctx context.Context, th *v1alpha1.Tekton
 
 	th.Status.MarkApiDependenciesInstalled()
 
+	pvc, err := r.checkPVC(ctx, th, "tekton-hub-api")
+	if err != nil {
+		if !apierrors.IsNotFound(err) {
+			return err
+		}
+	}
+
+	if pvc != nil {
+		if !r.checkPVCOwnerRef(pvc, th) {
+			ownerRef := *metav1.NewControllerRef(th, th.GroupVersionKind())
+			pvc.SetOwnerReferences([]metav1.OwnerReference{ownerRef})
+
+			_, err := r.kubeClientSet.CoreV1().PersistentVolumeClaims(th.Spec.GetTargetNamespace()).Update(ctx, pvc, metav1.UpdateOptions{})
+			if err != nil {
+				return err
+			}
+		}
```

Comment:
why don't we move this also into the `checkPVCOwnerRef(pvc, th)` function.

Reply:
yeah we can move that, but then function name should be changed as well, because if we are changing the ownerRef of pvc then on the cluster we are updating the pvc and I guess `checkPVCOwnerRef` is just checking the if it has the `TektonHub` as ownerRef